homeassistant/components/tibber/sensor.py

```diff
@@ -8,6 +8,12 @@
 
 import aiohttp
 
+from homeassistant.components.recorder.models import StatisticData, StatisticMetaData
```

Comment:
Do we need to add the recorder to dependencies of tibber now?

Reply:
It is added to `after_dependencies`
Is that correct?

>Built-in integrations shall only specify other built-in integrations in after_dependencies.

https://developers.home-assistant.io/docs/creating_integration_manifest/#after-dependencies

Reply:
If the dependency isn't always required by the integration or if we need to handle a missing dependency gracefully in a config flow we use `after_dependencies`. If the dependency is always required we use `dependencies` .